### ISSUE TITLE
Icon does not work on windows

### ISSUE THREAD

**Issue by s-a** [mikaelbr/node-notifier#62]
The icon does not display on windows using jpeg format. It displays a blue default info icon instead. I am using windows 7 with this code https://github.com/mikaelbr/node-notifier#cross-platform-advanced-usage

**Comment by maciekmp** [comment]
I think that it is not because of jpeg format but because there is no way to set icon in baloon notification on Windows 7

**Comment by ErikBean** [comment]
@maciekmp I think this should be possible. I have successfully set the icon for the tray balloon using Electron running on windows 7 x64:
![screen shot 2015-11-09 at 1 11 23 pm](https://cloud.githubusercontent.com/assets/3444917/11046671/89aef364-86e3-11e5-82f0-93c7a0bf2372.png)
Using [Electron's Tray API](https://github.com/atom/electron/blob/master/docs/api/tray.md)
I'm not sure if it supports jpegs, but I got it to work with [NativeImages](https://github.com/atom/electron/blob/master/docs/api/native-image.md) created from .png files and base64 data URIs. The APIs that Electron's NativeImage class uses should be available to node-notifier, is that correct? Our application uses custom tray balloon icons, so I'd like to know about any bugs on platforms I haven't tested, etc.
